### ISSUE TITLE
server-side deleted articles are not deleted on android

### ISSUE THREAD

**Issue by silberzwiebel** [wallabag/android-app#768]
### Actual behaviour
- I deleted articles with the webapp. After syncing the android app, the deleted articles are still in the list. I tried all synchronisation methods. Setting up the android app on a new device did not download the deleted articles neither do they appear somewhere in the webapp.

### Expected behaviour
- The deleted article is removed from the list in the android app.

# Steps to reproduce the issue
1. wallabag some random article
2. synchronize your android app
3. delete the random article using the webapp
4. synchronize your android app: the article is still there

# Environment details
- **wallabag app version**: 2.1.0
- **wallabag app installation source** (e.g. Gplay, F-Droid, manual): f-droid
- **Android OS version**: 4.2.2
- **Android ROM** (e.g. stock, LineageOS, SlimRom,…): inkbook's android (e.g., no google play services)
- **Android hardware**: [inkbook prime](https://www.inkbook.eu/inkbook-prime)
- **wallabag server version**: 2.3.6
- **Do you have Two-Factor-Authentication enabled?**: no

# Your experience with wallabag Android app
Thanks a lot for the whole wallabag experience! It makes me read the web with self-hosted, open-source services. Many thanks :)!

**Comment by Strubbl** [comment]
Thanks a lot for your bug report. I can reproduce the problem on my device.

**Comment by silberzwiebel** [comment]
No, that doesn't work either. I had a look at the logs with `adb shell logcat` and saw that the app seems to report that an article is existing although it clearly isn't:
```
V/Updater ( 5762): performSweep() articleID: 989, exists: true
```

Opening `https://mywallabag.tld/view/989` gives me the 404 page. The same is true for a range of other article IDs (somehow I managed to accidentally add the same article 10 times).

(sorry for the late reply, didn't get an e-mail for the notification in your edit)

**Comment by Strubbl** [comment]
maybe that is a bug in wallabag server then (if the API reports a false status). or in the https://github.com/di72nn/wallabag-api-wrapper from @di72nn

**Comment by silberzwiebel** [comment]
Eventually, the deleted articles were removed from the list. But I don't know how. Maybe, they have to be on top of the list?
